internal/lsp/signature.go

```diff
@@ -0,0 +1,33 @@
+package lsp
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/terraform-ls/internal/mdplain"
+	lsp "github.com/hashicorp/terraform-ls/internal/protocol"
+)
+
+func ToSignatureHelp(signature *lang.FunctionSignature) *lsp.SignatureHelp {
+	if signature == nil {
+		return nil
+	}
+
+	parameters := make([]lsp.ParameterInformation, 0)
+	for _, p := range signature.Parameters {
+		parameters = append(parameters, lsp.ParameterInformation{
+			Label:         p.Name,
+			Documentation: mdplain.Clean(p.Description.Value),
+		})
+	}
+
+	return &lsp.SignatureHelp{
+		Signatures: []lsp.SignatureInformation{
```

Comment:
Thinking out loud - what is even the use case for multiple signatures in the spec? 🤔

Reply:
That feature might be required for [function overloads](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads).

Reply:
I see, so not something we'd ever need in HCL then.